### DISCUSSION TITLE
Change download location

### DISCUSSION THREAD

**Discussion by paulliy** [pear-devs/pear-desktop#188] [Ideas]
I just want to ask if you can add something that allows you to change download location like from downloads to music for example. If there is already something like that then could you please show me. Thanks

**Comment by semvis123** [comment]
It is already possible, see #166 for more details

**Reply by Araxeus** [reply]
#207 There is now a download folder chooser in `downloader` plugin options

**Comment by inson1** [comment]
@paulliy May you close this discussion, if its resolved? :)
